src/main/java/xyz/aadev/generitech/common/tileentities/machines/TileEntityPulverizer.java

```diff
@@ -241,8 +281,12 @@ public void burnTime(int i) {
         }
     }
 
+
     @Override
     public void update() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The Cyclomatic Complexity of this method "update" is 43 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.aadev.xyz/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)